forage-android/src/main/java/com/joinforage/forage/android/core/services/forageapi/network/error/payload/ErrorPayload.kt

```diff
@@ -44,7 +44,7 @@ internal abstract class ErrorPayload(
                     return RosettaErrorResponse(jsonErrorResponse)
                 }
                 else -> {
-                    throw UnexpectedResponseError(jsonErrorResponse.toString())
+                    throw UnknownForageFailureResponse(jsonErrorResponse.toString())
```

Comment:
The new name more precisely describes that this exception is thrown when we there is a response that we believe is Forage-defined that we are unable to parse